src/wazuh_db/wdb_integrity.c

```diff
@@ -322,6 +323,34 @@ void wdbi_update_completion(wdb_t * wdb, wdb_component_t component, long timesta
     }
 }
 
+/**
+ * @brief This methods updates only the "last_completion" column.
```

Comment:
I think we can avoid the _"only"_ here. And the reference to other columns not modified

Reply:
Fixed!